### PR TITLE
1934 radio button tests

### PR DESCRIPTION
**Pull request by ElenaBokova**

### CODE REVIEW

**Comment by mf-code** [comment]
[5764] - 2lls8mE23JLz

**Comment by mf-code** [comment]
[5764] - wketPu9jPKYR

**Comment by mf-code** [comment]
[5764] - ToP5U5gR6smg

**Comment by mf-code** [comment]
[5764] - 94aJa0SYvoCi

**Comment by mf-code** [comment]
[5764] - 1XRnXOMF6y87

**Comment by mf-code** [comment]
[5764] - 4X60QeJYVj6f

**Comment by mf-code** [comment]
[5764] - 2JUI81a6BXch

**Comment by mf-code** [comment]
[5764] - 2OU7P2HhZO3l

**Comment by mf-code** [comment]
[5764] - utdTAwjzAkiO

**Comment by mf-code** [comment]
[5764] - 6EL7Q96O8WsO

**Comment by mf-code** [comment]
[5764] - VZcbtcu0GUPQ

**Comment by mf-code** [comment]
[5764] - NbPQafQL9tch

**Comment by mf-code** [comment]
[5764] - e4oYjVj7jdFc

**Comment by mf-code** [comment]
[5764] - 55FOCKKbPNhR

**Comment by mf-code** [comment]
[5764] - openjdk8 - Allure report on Netlify: https://5f0841847141c13e10297d4f--musing-swartz-6fc060.netlify.app

**Comment by mf-code** [comment]
[5764] - openjdk10 - Allure report on Netlify: https://5f0842876069687e91ffec27--musing-swartz-6fc060.netlify.app

File: jdi-light-angular-tests/src/test/java/io/github/epam/angular/tests/elements/complex/RadioButtonTests.java [hunk 1]
```diff
@@ -20,15 +20,109 @@ public void before() {
...
+    @Test
+    public void nameBasicRadioButtonTest() {
+        radioButtonSection.firstBasicRadioButton.has().text("Option 1");
+        radioButtonSection.secondBasicRadioButton.has().text("Option 2");
+    }
+
+    @Test
+    public void onlyOneOptionOneButtonEnabledTest() {
+        String enableBasicButton = "mat-radio-button mat-accent cdk-focused cdk-mouse-focused mat-radio-checked";
+        String disableBasicButton = "mat-radio-button mat-accent";
```
Comment (ViktarR) [inline jdi-light-angular-tests/src/test/java/io/github/epam/angular/tests/elements/complex/RadioButtonTests.java:47]: Could you please move this two variables to class variables, you duplicate them in two methods.

File: jdi-light-angular-tests/src/test/java/io/github/epam/angular/tests/elements/complex/RadioButtonTests.java [hunk 2]
```diff
@@ -20,15 +20,109 @@ public void before() {
...
+        radioButtonSection.autumnNGModelRadioButton.click();
+        radioButtonSection.favoriteSeason.has().text("Your favorite season is: Autumn");
+    }
+
+
+    @Test
+    public void onlyOneWinterButtonEnabledTest() {
+        String enableSeasonButton = "mat-radio-button example-radio-button mat-accent cdk-focused cdk-mouse-focused mat-radio-checked";
+        String disableSeasonButton = "mat-radio-button example-radio-button mat-accent";
+        radioButtonSection.winterNGModelRadioButton.click();
```
Comment (ViktarR) [inline jdi-light-angular-tests/src/test/java/io/github/epam/angular/tests/elements/complex/RadioButtonTests.java:88]: Could you please move this two variables to class variables, you use them in four methods.

**Comment by mf-code** [comment]
[5778] - IxxwwvWskqlC

**Comment by mf-code** [comment]
[5778] - A00b1c0sb6DF

**Comment by mf-code** [comment]
[5778] - y1JLFzPHibAK

**Comment by mf-code** [comment]
[5778] - pDoYIlJxcSS2

**Comment by mf-code** [comment]
[5778] - ciY7itcKkDVz

**Comment by mf-code** [comment]
[5778] - ViVOn5UJec0Q

**Comment by mf-code** [comment]
[5778] - ZSANhimX8bmq

**Comment by mf-code** [comment]
[5778] - ilZEhs99t3mp

**Comment by mf-code** [comment]
[5778] - TeCwdKhRlAFF

**Comment by mf-code** [comment]
[5778] - zuCfmyIm0qX9

**Comment by mf-code** [comment]
[5778] - jLf8tf2PmfJh

**Comment by mf-code** [comment]
[5778] - 1HMcE68lRFFx

**Comment by mf-code** [comment]
[5779] - EY15IS8mcyB3

**Comment by mf-code** [comment]
[5779] - dDNYQnSfrTRV

**Comment by mf-code** [comment]
[5778] - GLI1lK6gO3he

**Comment by mf-code** [comment]
[5779] - NXz6FtdL6aJH

**Comment by mf-code** [comment]
[5779] - 6SQHpaM0xca5

**Comment by mf-code** [comment]
[5778] - P8pLoVf6o1nW

**Comment by mf-code** [comment]
[5779] - c5uUyeTiztON

**Comment by mf-code** [comment]
[5779] - OXwwEMi9mj0b

**Comment by mf-code** [comment]
[5778] - openjdk8 - Allure report on Netlify: https://5f0c453dfa9b882a53b7462e--musing-swartz-6fc060.netlify.app

**Comment by mf-code** [comment]
[5779] - A78CU8nudCU4

**Comment by mf-code** [comment]
[5778] - openjdk10 - Allure report on Netlify: https://5f0c46254f8200f2eecfe9df--musing-swartz-6fc060.netlify.app

**Comment by mf-code** [comment]
[5779] - 4lkzpW3wlon4

**Comment by mf-code** [comment]
[5779] - zO2nl91mDpsY

**Comment by mf-code** [comment]
[5779] -

**Comment by mf-code** [comment]
[5779] - uECAYFtMgS1s

**Comment by mf-code** [comment]
[5779] - b4ZKmSpJvpYz

**Comment by mf-code** [comment]
[5779] - CQycy3BaaC9f

**Comment by mf-code** [comment]
[5780] - CGcAUFMIOz8Z

**Comment by mf-code** [comment]
[5779] - El6zqVQHQCv1

**Comment by mf-code** [comment]
[5780] - 3eP72r4HrDpz

**Comment by mf-code** [comment]
[5780] - ICfXrqxYkY60

**Comment by mf-code** [comment]
[5779] - openjdk8 - Allure report on Netlify: https://5f0c4b2f4f82001e16cfeaa4--musing-swartz-6fc060.netlify.app

**Comment by mf-code** [comment]
[5780] -

**Comment by mf-code** [comment]
[5780] - Uzrh546p4e3k

**Comment by mf-code** [comment]
[5779] - openjdk10 - Allure report on Netlify: https://5f0c4c0d84abef9d7e56f6dc--musing-swartz-6fc060.netlify.app

**Comment by mf-code** [comment]
[5780] - TiZUml49Hyr0

**Comment by mf-code** [comment]
[5780] - ynvLq1UCyCbp

**Comment by mf-code** [comment]
[5780] - QTWLXyjc6HxP

**Comment by mf-code** [comment]
[5780] - jT9qLhtbYWb1

**Comment by mf-code** [comment]
[5780] - VCJeMelFMbyi

**Comment by mf-code** [comment]
[5780] - zO8Yw1RiOsNe

**Comment by mf-code** [comment]
[5780] - xyXvdI3fjjJG

**Comment by mf-code** [comment]
[5780] - LLITouqZntit

**Comment by mf-code** [comment]
[5780] - X1sAUk8WuSRo

**Comment by mf-code** [comment]
[5780] - openjdk8 - Allure report on Netlify: https://5f0c52075b768ed4e760cbb8--musing-swartz-6fc060.netlify.app

**Comment by mf-code** [comment]
[5780] - openjdk10 - Allure report on Netlify: https://5f0c52b04f82005909cfeb06--musing-swartz-6fc060.netlify.app

File: jdi-light-angular-tests/src/main/java/io/github/com/pages/sections/RadioButtonSection.java [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+    @UI("#favorite-season-spring")
+    public static Button springNGModelRadioButton;
+
+    @UI("#favorite-season-summer")
+    public static Button summerNGModelRadioButton;
+
+    @UI("#favorite-season-autumn")
+    public static Button autumnNGModelRadioButton;
+
+    @UI("/html/body/div[1]/div[2]/div/radio-ng-model-example/div")
```
Comment (gbuh) [inline jdi-light-angular-tests/src/main/java/io/github/com/pages/sections/RadioButtonSection.java:34]: This locator is unstable.
Please replace this or enter an id for this element.

File: jdi-light-angular-tests/src/main/java/io/github/com/pages/sections/RadioButtonSection.java [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+    public static Text favoriteSeason;
+
+    public static String enableSeasonButton = "mat-radio-button example-radio-button mat-accent cdk-focused cdk-mouse-focused mat-radio-checked";
```
Comment (gbuh) [inline jdi-light-angular-tests/src/main/java/io/github/com/pages/sections/RadioButtonSection.java:37]: The class name is too long. Maybe it's better to use a significant part of the class name and Matcher in method attr(String attrName, Matcher<String> condition)

**Comment by mf-code** [comment]
[5786] - 56v3rvqBenFb

**Comment by mf-code** [comment]
[5786] - dciRHl0vJT1y

**Comment by mf-code** [comment]
[5786] - WA0lbaMpL7kD

**Comment by mf-code** [comment]
[5786] - hYUHvwggJ8dC

**Comment by mf-code** [comment]
[5786] - LxCX6VOEyUOP

**Comment by mf-code** [comment]
[5786] - ZGxqQYNfGph3

**Comment by mf-code** [comment]
[5786] - 1r4CjJWilg0C

**Comment by mf-code** [comment]
[5786] - Ap3MSHIRhxSt

**Comment by mf-code** [comment]
[5786] - rOHKi6Wm7lKe

**Comment by mf-code** [comment]
[5786] - HckouOU5UBCc

**Comment by mf-code** [comment]
[5786] - jgpkc5TSKkDQ

**Comment by mf-code** [comment]
[5786] - CvmX7Bg12WU6

**Comment by mf-code** [comment]
[5786] - uTT4ThfuPPLB

**Comment by mf-code** [comment]
[5786] - 8zWHDFJPchOb

**Comment by mf-code** [comment]
[5786] - openjdk8 - Allure report on Netlify: https://5f0d810e89a086ce87c70b45--musing-swartz-6fc060.netlify.app

**Comment by mf-code** [comment]
[5786] - openjdk10 - Allure report on Netlify: https://5f0d81d5c012300b6048d51b--musing-swartz-6fc060.netlify.app

File: jdi-light-angular-tests/src/main/java/io/github/com/pages/sections/RadioButtonSection.java [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+    public static Button autumnNGModelRadioButton;
+
+    @UI("#your-favorite-season-text")
+    public static Text favoriteSeason;
+
+    public static String enableSeasonButton = ".* cdk-focused cdk-mouse-focused mat-radio-checked";
+    public static String disableSeasonButton = "mat-radio-button example-radio-button mat-accent";
+
+    public static String enableBasicButton = ".* cdk-focused cdk-mouse-focused mat-radio-checked";
+    public static String disableBasicButton = "mat-radio-button mat-accent";
```
Comment (ViktarR) [inline jdi-light-angular-tests/src/main/java/io/github/com/pages/sections/RadioButtonSection.java:41]: Seems that we are not on the same page. Could you please move that variables to the test class(RadioButtonTests), they are part of test not the section(RadioButtonSection).
And the access modifier should be private static.

**Comment by mf-code** [comment]
[5793] - wU6HVmGJispC

**Comment by mf-code** [comment]
[5793] - WnYpmdSz2Bnr

**Comment by mf-code** [comment]
[5793] - w04mgO34PsAG

**Comment by mf-code** [comment]
[5793] - o6lyjgWFmoAQ

**Comment by mf-code** [comment]
[5793] - BbZAuZ0JiJ8h

**Comment by mf-code** [comment]
[5793] - mn2iWEbzIaLL

**Comment by mf-code** [comment]
[5793] - uLWCNDx3R50f

**Comment by mf-code** [comment]
[5793] - Pg8YbkwHFhE1

**Comment by mf-code** [comment]
[5793] - 18h2Bu99aBL7

**Comment by mf-code** [comment]
[5793] - ys5EflPn0Pqx

**Comment by mf-code** [comment]
[5793] -

**Comment by mf-code** [comment]
[5793] - oz4RuSjWw48N

**Comment by mf-code** [comment]
[5793] - G9XQQXV2zkGH

**Comment by mf-code** [comment]
[5793] - BbSP4Qx7YDAj

**Comment by mf-code** [comment]
[5793] - openjdk8 - Allure report on Netlify: https://5f0e00191c4bcaf9feb2f0bd--musing-swartz-6fc060.netlify.app

**Comment by mf-code** [comment]
[5793] - openjdk10 - Allure report on Netlify: https://5f0e012438ed48ad2b625f8a--musing-swartz-6fc060.netlify.app

File: jdi-light-angular-tests/src/test/java/io/github/epam/angular/tests/elements/complex/RadioButtonTests.java [hunk 6]
```diff
@@ -1,34 +1,125 @@
...
+    @Test
+    public void onlyOneAutumnButtonEnabledTest() {
+        radioButtonSection.autumnNGModelRadioButton.click();
+        radioButtonSection.winterNGModelRadioButton.has().attr("class", disableSeasonButton);
+        radioButtonSection.springNGModelRadioButton.has().attr("class", disableSeasonButton);
+        radioButtonSection.summerNGModelRadioButton.has().attr("class", disableSeasonButton);
+        radioButtonSection.autumnNGModelRadioButton.has().attr("class", matchesRegex(enableButton));
+    }
+
 }
```
Comment (ViktarR) [inline jdi-light-angular-tests/src/test/java/io/github/epam/angular/tests/elements/complex/RadioButtonTests.java:125]: Could you please pay attention to count of empty rows between tests, somewhere you paste two rows. Could you please, delete extra rows.

File: jdi-light-angular-tests/src/test/java/io/github/epam/angular/tests/elements/complex/RadioButtonTests.java [hunk 7]
```diff
@@ -1,34 +1,125 @@
...
+        radioButtonSection.autumnNGModelRadioButton.has().attr("class", disableSeasonButton);
     }
+
+    @Test
+    public void onlyOneAutumnButtonEnabledTest() {
+        radioButtonSection.autumnNGModelRadioButton.click();
+        radioButtonSection.winterNGModelRadioButton.has().attr("class", disableSeasonButton);
+        radioButtonSection.springNGModelRadioButton.has().attr("class", disableSeasonButton);
+        radioButtonSection.summerNGModelRadioButton.has().attr("class", disableSeasonButton);
+        radioButtonSection.autumnNGModelRadioButton.has().attr("class", matchesRegex(enableButton));
```
Comment (ViktarR) [inline jdi-light-angular-tests/src/test/java/io/github/epam/angular/tests/elements/complex/RadioButtonTests.java:122]: Could you please move "class" to the variables, the same as disableBasicButton.

File: jdi-light-angular-tests/src/test/java/io/github/epam/angular/tests/elements/complex/RadioButtonTests.java [hunk 8]
```diff
@@ -1,34 +1,125 @@
...
+import static org.hamcrest.Matchers.containsString;
+import static org.hamcrest.Matchers.matchesRegex;
 import static org.testng.Assert.assertEquals;
 
 public class RadioButtonTests extends TestsInit {
 
+    private static String enableButton = ".* cdk-focused cdk-mouse-focused mat-radio-checked";
+    private static String disableSeasonButton = "mat-radio-button example-radio-button mat-accent";
+    private static String disableBasicButton = "mat-radio-button mat-accent";
+
```
Comment (gbuh) [inline jdi-light-angular-tests/src/test/java/io/github/epam/angular/tests/elements/complex/RadioButtonTests.java:22]: Please remove all double blank lines in classes. After formatting, there should be only one blank line between code blocks.

**Comment by mf-code** [comment]
[5795] - 5gyycQAxnq6T

**Comment by mf-code** [comment]
[5795] - h21OXLWQayzB

**Comment by mf-code** [comment]
[5795] - 3xczXKde5u8O

**Comment by mf-code** [comment]
[5795] - mIaadql63JNv

**Comment by mf-code** [comment]
[5795] - BJJGC6gmFxQ3

**Comment by mf-code** [comment]
[5795] - 2HQrfURleDEM

**Comment by mf-code** [comment]
[5795] - W9BmPhTftcD1

**Comment by mf-code** [comment]
[5795] - 19pK5jOW0V1X

**Comment by mf-code** [comment]
[5795] - XWQPk4ps19P8

**Comment by mf-code** [comment]
[5795] - Udk2s14qiYOV

**Comment by mf-code** [comment]
[5795] - zChvK1iyXz6Y

**Comment by mf-code** [comment]
[5795] - Dk5hqGFMrKfJ

**Comment by mf-code** [comment]
[5795] - DwmY6tzzluIK

**Comment by mf-code** [comment]
[5796] - TsvtGTYL7Zew

**Comment by mf-code** [comment]
[5796] - XsFiScs4saJd

**Comment by mf-code** [comment]
[5795] - 5TTu9G833TB5

**Comment by mf-code** [comment]
[5796] - yd1dWXDPLO2s

**Comment by mf-code** [comment]
[5796] - nNnjZQPPJc5j

**Comment by mf-code** [comment]
[5795] - openjdk8 - Allure report on Netlify: https://5f0eba8b5550e9c5c5f5d0d1--musing-swartz-6fc060.netlify.app

**Comment by mf-code** [comment]
[5796] - B2YjdwyCiewI

**Comment by mf-code** [comment]
[5795] - openjdk10 - Allure report on Netlify: https://5f0ebb70f5aff019ebab8935--musing-swartz-6fc060.netlify.app

**Comment by mf-code** [comment]
[5796] - LtvEhA6ozyhy

**Comment by mf-code** [comment]
[5796] - ljFb2bgkE8Kt

**Comment by mf-code** [comment]
[5796] - GIRql2Bg9Bpl

**Comment by mf-code** [comment]
[5796] - Om3VkNh8wKRT

**Comment by mf-code** [comment]
[5796] - 1vR3X7f4ez1Z

**Comment by mf-code** [comment]
[5796] - icFmPzLYoV31

**Comment by mf-code** [comment]
[5796] - fkOk8MrSKHt1

**Comment by mf-code** [comment]
[5796] - 1enj99XMXLOM

**Comment by mf-code** [comment]
[5796] - 3j9AWLzHl5L8

**Comment by mf-code** [comment]
[5796] - openjdk8 - Allure report on Netlify: https://5f0ec4b954d53872acadd475--musing-swartz-6fc060.netlify.app

**Comment by mf-code** [comment]
[5796] - openjdk10 - Allure report on Netlify: https://5f0ec5b2980bda8632497871--musing-swartz-6fc060.netlify.app

File: jdi-light-angular-tests/src/test/java/io/github/epam/angular/tests/elements/complex/RadioButtonTests.java [hunk 9]
```diff
@@ -1,34 +1,121 @@
...
 import static io.github.com.StaticSite.angularPage;
-import static io.github.com.pages.AngularPage.basicRadioButtons;
-import static io.github.com.pages.AngularPage.firstBasicRadioButton;
-import static io.github.com.pages.AngularPage.secondBasicRadioButton;
+import static io.github.com.pages.AngularPage.*;
 import static io.github.epam.site.steps.States.shouldBeLoggedIn;
+import static org.apache.tika.mime.MediaType.text;
+import static org.hamcrest.Matchers.containsString;
+import static org.hamcrest.Matchers.matchesRegex;
 import static org.testng.Assert.assertEquals;
```
Comment (gbuh) [inline jdi-light-angular-tests/src/test/java/io/github/epam/angular/tests/elements/complex/RadioButtonTests.java:16]: There are many unused import statements in the class.
Please format file (delete unused import statements).

File: jdi-light-angular-tests/src/main/java/io/github/com/pages/sections/RadioButtonSection.java [hunk 10]
```diff
@@ -0,0 +1,37 @@
+package io.github.com.pages.sections;
+
+import com.epam.jdi.light.elements.pageobjects.annotations.locators.Css;
+import com.epam.jdi.light.elements.pageobjects.annotations.locators.UI;
+import com.epam.jdi.light.ui.html.elements.common.Button;
+import com.epam.jdi.light.ui.html.elements.common.Text;
+import com.epam.jdi.light.ui.html.elements.complex.RadioButtons;
+
+
```
Comment (gbuh) [inline jdi-light-angular-tests/src/main/java/io/github/com/pages/sections/RadioButtonSection.java:9]: Could you please, delete extra row.

**Comment by mf-code** [comment]
[5801] - 0bSfWSBRBD10

**Comment by mf-code** [comment]
[5801] - qFrbIZoTmS84

**Comment by mf-code** [comment]
[5801] - cPFpXNi7mMLS

**Comment by mf-code** [comment]
[5801] - OZGsspB20mxD

**Comment by mf-code** [comment]
[5801] - 0nkmrImOlP00

**Comment by mf-code** [comment]
[5801] - 2lj3KPnJNexZ

**Comment by mf-code** [comment]
[5801] - I9sO5iM06NDN

**Comment by mf-code** [comment]
[5801] - U3qwx7lMuyCn

**Comment by mf-code** [comment]
[5801] - Qm49hhsymDaN

**Comment by mf-code** [comment]
[5801] - 1XuOSccK1NPk

**Comment by mf-code** [comment]
[5801] - DCnZwexySlfT

**Comment by mf-code** [comment]
[5801] - gcOCl9KUjmGb

**Comment by mf-code** [comment]
[5802] - LULDZE4GLBOb

**Comment by mf-code** [comment]
[5801] - 4zRv4aMcroNf

**Comment by mf-code** [comment]
[5802] - jnO9DMXGqzjn

**Comment by mf-code** [comment]
[5801] - j3UklamU4yYZ

**Comment by mf-code** [comment]
[5802] - NQ8EJ9hsKtgf

**Comment by mf-code** [comment]
[5802] - MYASotC8iLT1

**Comment by mf-code** [comment]
[5801] - openjdk8 - Allure report on Netlify: https://5f0f22e335ac4b29ea6be796--musing-swartz-6fc060.netlify.app

**Comment by mf-code** [comment]
[5802] - K3dEzihIQN1H

**Comment by mf-code** [comment]
[5802] - D5DzyyIZoRfB

**Comment by mf-code** [comment]
[5801] - openjdk10 - Allure report on Netlify: https://5f0f23ccd32607b2d90f50ef--musing-swartz-6fc060.netlify.app

**Comment by mf-code** [comment]
[5802] - 0KUCVu2oNBPW

**Comment by mf-code** [comment]
[5802] - ffldtttc8epj

**Comment by mf-code** [comment]
[5802] - tnBVRg6GWB1n

**Comment by mf-code** [comment]
[5802] - TEQxaiQzQ8VA

**Comment by mf-code** [comment]
[5802] - ewTxVlPz20Ee

**Comment by mf-code** [comment]
[5802] - BV3d1QZJUaTp

**Comment by mf-code** [comment]
[5802] -

**Comment by mf-code** [comment]
[5802] - oaaB1M4Q3jCG

**Comment by mf-code** [comment]
[5802] - openjdk8 - Allure report on Netlify: https://5f0f29caccd72630d132a97f--musing-swartz-6fc060.netlify.app

**Comment by mf-code** [comment]
[5802] - openjdk10 - Allure report on Netlify: https://5f0f2a1aa777d6df4b2d3f54--musing-swartz-6fc060.netlify.app

**Comment by mf-code** [comment]
[5805] - s3YnAKzmlLKK

**Comment by mf-code** [comment]
[5805] - lss6Zt6A7oMv

**Comment by mf-code** [comment]
[5805] - WbDRMDaQ0C9q

**Comment by mf-code** [comment]
[5805] - TG2HZccVLGKw

**Comment by mf-code** [comment]
[5805] - A9Z8Blw5ygY9

**Comment by mf-code** [comment]
[5805] - uFIcuAwFP0jN

**Comment by mf-code** [comment]
[5805] - PtXyHsjr3Gh6

**Comment by mf-code** [comment]
[5805] - 6lK4XBCAawQW

**Comment by mf-code** [comment]
[5805] - AzNEA9UUPOfn

**Comment by mf-code** [comment]
[5805] - llR7rkNYMjI4

**Comment by mf-code** [comment]
[5805] - zI6xf68odTdM

**Comment by mf-code** [comment]
[5805] - SofTwNwCqK62

**Comment by mf-code** [comment]
[5805] - JIQLX3gqmNss

**Comment by mf-code** [comment]
[5805] - dfqiujINIoVt

**Comment by mf-code** [comment]
[5805] - openjdk8 - Allure report on Netlify: https://5f0f43e62ad4ba9f69692a45--musing-swartz-6fc060.netlify.app

**Comment by mf-code** [comment]
[5805] - openjdk10 - Allure report on Netlify: https://5f0f44e11f48aea0bbfb2bb0--musing-swartz-6fc060.netlify.app

**Pull request merged**